src/wallet/coinselection.cpp

```diff
@@ -384,6 +400,17 @@ CAmount GetSelectionWaste(const std::set<CInputCoin>& inputs, CAmount change_cos
     return waste;
 }
 
+CAmount GenerateChangeTarget(CAmount payment_value, FastRandomContext& rng)
+{
+    if (payment_value < CHANGE_LOWER / 2) {
```

Comment:
:rotating_light:  

This crashes the node

```suggestion
    if (payment_value <= CHANGE_LOWER / 2) {
```

```
bitcoin-qt: ./random.h:204: uint64_t FastRandomContext::randrange(uint64_t): Assertion `range' failed.
Aborted (core dumped)

Reply:
Oops, fixed.